app/views/layouts/application.html.erb

```diff
@@ -104,15 +104,21 @@
       </div>
     <% end %>
     <%= yield %>
+    <% if FeatureFlag.enabled?(:runtime_banner) || Rails.env.test? %>
```

Comment:
Added the `Rails.env.text?` bit so the banner will be visible in E2E tests. This entire condition will be removed when the `FeatureFlag` is gone